packages/mobile/src/verify/saga.ts

```diff
@@ -898,23 +887,27 @@ const sagas: Array<[string, any]> = [
   [revoke.type, revokeSaga],
 ]
 
+export function* watchVerificationEvents() {
+  for (const [actionType, saga] of sagas) {
+    yield takeEvery(actionType, createErrorHandler(saga, actionType))
```

Comment:
Nit: I think `createErrorHandler` could be named something more descriptive like `callAndHandleErrors`. It wasn't obvious to me that this function would `call` the sagas that were being passed to it...I assumed that it was only doing something around handling errors